src/tokens/safe.tokenlist.json

```diff
@@ -3,18 +3,50 @@
   "timestamp": "2023-03-15T11:00:00+00:00",
   "version": {
     "major": 1,
-    "minor": 0,
+    "minor": 1,
     "patch": 0
   },
   "keywords": ["default", "list", "safe"],
   "tokens": [
+    {
+      "symbol": "COW",
+      "name": "CoW Protocol Token",
+      "address": "0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab",
+      "decimals": 18,
+      "chainId": 1,
+      "logoURI": "https://safe-transaction-assets.safe.global/tokens/logos/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB.png"
```

Comment:
it's confusing that cow token logos on mainnet and gc are so different. Is this intended?

Reply:
Very weird indeed.

I am not sure how we pulled those into our assets. @Uxio0 Do you know more about this? Could we update them to the same logo?